gadget/ondisk.go

```diff
@@ -216,6 +216,7 @@ func onDiskVolumeFromPartitionTable(ptable sfdiskPartitionTable) (*OnDiskVolume,
 			Label:      bd.Label,
 			Type:       vsType,
 			Filesystem: bd.FSType,
+			ID:         strings.ToUpper(p.UUID),
```

Comment:
maybe worth to update VolumeStructure.ID documentation

Reply:
done